probert/tests/test_multipath.py

```diff
@@ -0,0 +1,76 @@
+import mock
+import subprocess
+
+from probert import multipath
+from probert.tests.helpers import CiTestCase
+
+
+class TestMultipath(CiTestCase):
+
+    @mock.patch('probert.multipath.subprocess.run')
+    def test_multipath_show_paths(self, m_run):
+        mp_out = " ".join([self.random_string() for x in range(0, 8)])
+        cp = subprocess.CompletedProcess(args = ['foo'], returncode = 0,
```

Comment:
Not super-important, but FYI the spacing around the `=`s for keyword args here is non-standard per [PEP-8](https://www.python.org/dev/peps/pep-0008/#other-recommendations):

> Don't use spaces around the = sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter.

Reply:
I'll fix.  Looks like we should add pep8 to make lint target

Reply:
I've filed #63 for that